cosmos/testing/integration/rpc/eth_test.go

```diff
@@ -45,76 +49,169 @@ func TestRpc(t *testing.T) {
 
 var _ = Describe("Network", func() {
 	var client *ethclient.Client
+	var wsClient *ethclient.Client
```

Comment:
lgtm, but need to fix everything up as you said before.

Reply:
bless